webapp/lib/model.g.dart

```diff
@@ -85,7 +88,7 @@ class Message {
     if (data == null) return null;
     return (modelObj ?? Message())
       ..direction = MessageDirection.fromString(data['direction'] as String) ?? MessageDirection.Out
-      ..datetime = DateTime_fromData(data['datetime']) ?? DateTime.now()
+      ..datetime = DateTime_fromData(data['datetime'])
```

Comment:
Is this because `datetime` is now expected to be mandatory as a field?

Reply:
@marianamarasoiu @lukechurch Correct. All fields will eventually be mandatory, but it will take some time to get there. When new field are introduced (e.g. MessageStatus), they will have a default value until they can be fully plumbed through the back end and all existing data migrated to have a value. Once that happens, the default value can be removed.